privacyidea/static/components/config/views/config.system.html

```diff
@@ -144,6 +144,32 @@
                                     <span translate>Help on Tokengroups</span></a></li>
                         </ul>
                     </li>
+                    <li role="presentation"
+                        ng-class="{ active: $state.includes('config.serviceid')}"
+                        ng-show="checkRight('serviceid_list')">
+                        <a ui-sref="config.serviceid">
+                            <span class="glyphicon glyphicon-cutlery"></span>
+                            <span translate>Service IDs</span>
+                        </a>
+                        <ul ng-show="$state.includes('config.serviceid')"
+                            class="nav nav-pills nav-stacked pi-submenu">
+                            <li role="presentation"
+                                ng-class="{ active: $state.includes('config.serviceid.list')}">
+                                <a ui-sref="config.serviceid.list"
+                                        translate>List Service ID definitions</a></li>
+                            <li role="presentation"
+                                ng-class="{ active: $state.includes('config.serviceid.add')}"
+                                ng-show="checkRight('serviceid_add')">
+                                <a ui-sref="config.serviceid.add" translate>
+                                    New Service ID definition</a></li>
+                            <hr>
+                            <li role="presentation">
+                                <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                                   privacyideaVersionNumber+'/configuration/serviceids.html'}}">
```

Comment:
Do You plan to add this site to the documentation? Otherwise we should point this to another (existing) read-the-docs site about service ids.

Reply:
I think it makes sense to add such docs.
Will do.

Reply:
Done in ed4d272c